test/integration_test/utils/constants.go

```diff
@@ -36,7 +39,11 @@ const (
 	// DefaultValidateUninstallTimeout is a default timeout for uninstall validation
 	DefaultValidateUninstallTimeout = 15 * time.Minute
 	// DefaultValidateUninstallRetryInterval is a default retry interval for uninstall validation
-	DefaultValidateUninstallRetryInterval = 10 * time.Second
+	DefaultValidateUninstallRetryInterval = 20 * time.Second
+	// DefaultValidateStorkTimeout is a default timeout for stork validation
+	DefaultValidateStorkTimeout = 60 * time.Minute
```

Comment:
Let's keep this 10 min. 1 hour seems too long.

Reply:
Changed `DefaultValidateStorkTimeout` to 10 mins